resources/sass/global/_helpers.scss

```diff
@@ -23,3 +23,7 @@
 .body--overflow-hidden {
 	overflow: hidden;
 }
+
+.display--none {
```

Comment:
Looks like this class isn't being used anywhere, but even if it was, bootstrap offers this as `.d-none`, so please remove this CSS rule 👍